src/lib/dto/v6/request/view-entry-create-request-v6.ts

```diff
@@ -0,0 +1,8 @@
+import { AbstractSyncableCreateRequestV6 } from "./abstract-syncable-create-request-v6";
+
+export interface ViewEntryCreateRequestV6 extends AbstractSyncableCreateRequestV6 {
+  viewSectionUuid: string;
```

Comment:
I'm pretty sure `viewSectionUuid` is nullable, so we should a question mark at the end of the property name.